README.md

```diff
@@ -38,7 +38,7 @@ or if you prefer storing the config elsewhere, in a yml file for example:
 
 ```ruby
 # config/initializers/better_html.rb
-BetterHtml.config = BetterHtml::Config.new(YAML.load(File.read('/path/to/.better-html.yml')))
+BetterHtml.config = BetterHtml::Config.new(YAML.unsafe_load_file('/path/to/.better-html.yml'))
```

Comment:
Alternatively could use the safe method style:

```suggestion
BetterHtml.config = BetterHtml::Config.new(YAML.load_file(file_path, permitted_classes: [Regexp]))
```
However, I assume most people are in control of the file if they are doing this, and other classes may need to be added to the argument. `unsafe_*` may freak people out a bit but will work.